app/webpack/javascripts/modules/Modules.AllocationFilterSubmit.js

```diff
@@ -1,8 +1,50 @@
 moj.Modules.AllocationFilterSubmit = {
   init: function () {
-    $('.allocation-filter-form')
-      .on('change', ':radio', function () {
-        $(this).closest('form').submit()
+    const reAllocationForm = document.querySelector('.allocation-filter-form')
```

Comment:
Could we rename this module to `moj.Modules.ReAllocationFilterSubmit` for clarity as it only appears to be used by the reallocation form?